pandas/core/frame.py

```diff
@@ -7390,20 +7390,26 @@ def f(s):
     def quantile(self, q=0.5, axis=0, numeric_only=True,
                  interpolation='linear'):
         """
-        Return values at the given quantile over requested axis.
+        Return value(s) at the given quantile over requested axis.
+
+        This function returns the Series of 'q' quantile value(s)
+        from the DataFrame, dividing data points into groups
+        along `axis` axis
+        In case of insufficient number of data points for clean division
+        into groups, specify `interpolation` scheme to implement.
 
         Parameters
         ----------
-        q : float or array-like, default 0.5 (50% quantile)
-            0 <= q <= 1, the quantile(s) to compute
-        axis : {0, 1, 'index', 'columns'} (default 0)
-            0 or 'index' for row-wise, 1 or 'columns' for column-wise
-        numeric_only : boolean, default True
+        q : float or array-like, default 0.5
```

Comment:
`array-like` is used for an object that implements the `numpy.array` api. I think just `list` would be better here.